src/api/content/content.service.ts

```diff
@@ -1,23 +1,9 @@
-// Sanity Client
-import { client } from '../_helpers/sanity-connector';
-
 // Queries
-import { mapStorylist, mapStorylistTeaser } from '../_utils/functions';
-import { fetchLandingPageContentQuery } from '../_queries/content.query';
-import { FetchLandingPageContentQueryResult } from '../sanity/types';
+import { fetchStorylistTeasers } from '../storylist/storylist.service';
 
 export async function fetchLandingPageContent() {
-	const result: FetchLandingPageContentQueryResult = await client.fetch(fetchLandingPageContentQuery);
-	const cards = [];
-	const previews = [];
-
-	for (const preview of result.previews) {
-		previews.push(await mapStorylist(preview));
-	}
-
-	for (const card of result.cards) {
-		cards.push(await mapStorylistTeaser(card));
-	}
+	const cards = await fetchStorylistTeasers();
+	const previews: [] = [];
```

Comment:
```suggestion
	const previews: StorylistTeaser[] = [];
```
Sería ese el tipo correcto haciendo el rastreo desde `app/content.service.ts`, no?

Bah, por qué no devolvemos el array vacío directamente con una aclaración de por qué hace falta seguir devolviendo el []?

Reply:
Es correcto que sea vacío (o sea, de tipo `Array<never>`) dado que el campo previews probablemente luego lo borremos del todo y la carga de la home tome una forma radicalmente diferente en el futuro inmediato. Agrego el detalle para que lo corrijamos en #996.